### PR TITLE
remove lingeringOnCommitWouldBeBeneficial and unused imports

### PR DESCRIPTION
**Pull request by sangreal**
Description...
1. remove lingeringOnCommitWouldBeBeneficial since it is not used and make the logic more complex
2. remove unused imports
### Checklist

- [ ] Documentation (if applicable)
- [x] Changelog

### CODE REVIEW

**Comment by sangreal** [comment]
> Looks good, thank you. Can you please regenerate Readme - to incorporate Changelog? Run `mvn asciidoc-template:build` or use Run configuration `README` in IDE. Ignore errors on submodules for the readme generation - they dont have readme template but report as errors when its not found.
@rkolesnev thanks for the review. README updated

File: CHANGELOG.adoc [hunk 1]
```diff
@@ -25,6 +25,9 @@ endif::[]
 * improvement: add multiple caches for accelerating available container count calculation （#667）
 * improvement: RecordContext now exposes lastFailureReason (#725)
 
+=== Fixes
+* fix: remove lingeringOnCommitWouldBeBeneficial and unused imports
+
```
Comment (rkolesnev) [inline CHANGELOG.adoc:30]: Could you please move this higher up - merge into the Fixes subsection above? And regenerate Readme again once its done.

**Review by rkolesnev — CHANGES_REQUESTED** [review]

File: CHANGELOG.adoc (hunk 1 above)
Comment (sangreal) [reply CHANGELOG.adoc]: @rkolesnev thanks for pointing out, changed accordingly.

**Pull request merged**